src/components/ContextualSaveBar/tests/ContextualSaveBar.test.tsx

```diff
@@ -2,7 +2,7 @@ import React from 'react';
 import {mount} from 'enzyme';
 import {mountWithAppProvider} from 'test-utilities/legacy';
 import ContextualSaveBar from '../ContextualSaveBar';
-import {FrameContext, createFrameContext} from '../../Frame';
```

Comment:
FrameProvider is inside `TestProvider` so we just need to pass our spies through